src/EventListener/DcaAjaxOperationsListener.php

```diff
@@ -202,7 +211,7 @@ private function getDefaultButtonCallback(string $name, string $table, array $se
                 return '';
             }
 
-            $value = $row[$settings['field']];
+            $value = (string) $row[$settings['field']];
```

Comment:
There might even be a better way than casting the database values to string to compare against the option value. But for my case it was working like that. what do you think?